crates/neon-sys/src/raw.rs

```diff
@@ -26,6 +43,11 @@ impl HandleScope {
 
 const ESCAPABLE_HANDLE_SCOPE_SIZE: usize = 32;
 
+/// A V8 `EscapableHandleScope`.
+///
+/// A `EscapableHandleScope` is like `HandleScope` but also allows us to push `Local` handles out
+/// to the previous `HandleScope`, preventing the `Local` value from being garbage collected when
```

Comment:
"preventing...from being garbage collected" isn't _quite_ the right way to say it, I think. Maybe something like, "allowing the `Local` value to remain rooted longer than the `HandleScope` where it was initially rooted." If you think that's too confusing and nit-picky, though, don't worry about it.